test/java/seedu/addressbook/commands/SortCommandTest.java

```diff
@@ -0,0 +1,41 @@
+package seedu.addressbook.commands;
+
+import org.junit.Test;
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.*;
+import seedu.addressbook.util.TestUtil;
+
+import java.util.*;
+
+import static org.junit.Assert.*;
+
+public class SortCommandTest {
+
+    @Test
+    public void execute_sortCommand_threePersons() {
+        SortCommand command = new SortCommand();
+
+        Person alice = TestUtil.generateTestPerson("Alice");
```

Comment:
Going further you can consider moving all the steps necessary for the test to run (or preconditions) to a setup method before beginning the tests; correspondingly can put all the tear down in a method after the tests.